hail/src/main/scala/is/hail/expr/ir/IR.scala

```diff
@@ -427,7 +427,7 @@ final case class SeqOp(i: Int, args: IndexedSeq[IR], aggSig: PhysicalAggSig) ext
 final case class CombOp(i1: Int, i2: Int, aggSig: PhysicalAggSig) extends IR
 final case class ResultOp(startIdx: Int, aggSigs: IndexedSeq[PhysicalAggSig]) extends IR
 final case class CombOpValue(i: Int, value: IR, aggSig: PhysicalAggSig) extends IR
-final case class AggStateValue(i: Int, aggSig: AggStateSig) extends IR
+final case class AggStateValue(i: Int, aggSig: PhysicalAggSig) extends IR
```

Comment:
I don't think this should be a PhysicalAggSig---the aggOp should have no bearing on how you serialize the state.

Reply:
yeah, had the same thought this morning. CombOpValue too, I think.